content/docs/topics/chart_best_practices/custom_resource_definitions.md

```diff
@@ -26,17 +26,37 @@ But there's a difference with CRDs.
 For a CRD, the declaration must be registered before any resources of that CRDs
 kind(s) can be used. And the registration process sometimes takes a few seconds.
 
-### Method 1: Separate Charts
-
-One way to do this is to put the CRD definition in one chart, and then put any
-resources that use that CRD in _another_ chart.
-
-In this method, each chart must be installed separately.
-
-### Method 2: Pre-install Hooks
-
-To package the two together, add a `pre-install` hook to the CRD definition so
-that it is fully installed before the rest of the chart is executed.
-
-Note that if you create the CRD with a `pre-install` hook, that CRD definition
-will not be uninstalled when `helm uninstall` is run.
+### Method 1: Let `helm` Do It For You
+
+With the arrival of Helm 3, we removed the old `crd-install` hooks for a more
+simple methodology. There is now a special directory called `crds` that you can
+create in your chart to hold your CRDs. These CRDs are not templated, but will
+be installed by default when running a `helm install` for the chart. If the CRD
+already exists, it will be skipped with a warning. If you wish to skip the CRD
+installation step, you can pass the `--skip-crds` flag.
+
+#### Some caveats (and explanations)
```

Comment:
I would like at least 2 maintainers to weigh in on the explanations in this section and make sure they are well-worded and cover everything they need to

Reply:
I validate that this is correct.

Reply:
yup